cellprofiler/modules/namesandtypes.py

```diff
@@ -958,6 +975,28 @@ def settings(self):
                 single_image.save_outlines, single_image.manual_rescale]
 
         return result
+    
+    def help_settings(self):
+        result = [
+            self.assignment_method,
+            self.single_load_as_choice,
+            self.matching_choice,
+            self.single_rescale,
+            self.assignments_count,
+            self.single_images_count,
+            self.manual_rescale,
+            self.process_as_3d,
+            self.x,
+            self.y,
+            self.z
+        ]
+
+        for assignment in self.assignments:
```

Comment:
This will result in duplicate documentation for multiple assignments. It's probably best to use the settings from `self.assignments[0]`.